src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -519,17 +508,11 @@ private static bool IsVersionSupported(WorkerLanguageVersionInfo info)
         {
             if (info?.Major == 3)
             {
-                switch (info?.Minor)
+                return (info?.Minor) switch
                 {
-                    case 9:
-                        // We currently do not support Python 3.9 on Windows.
-                        if (PlatformHelper.IsWindows) { return false; }
-                        else return true;
-                    case 8:
-                    case 7:
-                    case 6: return true;
-                    default: return false;
-                }
+                    9 or 8 or 7 or 6 => true,
```

Comment:
Seems like the pipeline is complaining the new syntax. Let's use the old fashion one.